homeassistant/components/withings/__init__.py

```diff
@@ -142,7 +141,7 @@ async def unregister_webhook(
         await hass.data[DOMAIN][entry.entry_id].async_unsubscribe_webhooks()
 
     async def register_webhook(
-        call_or_event_or_dt: ServiceCall | Event | datetime | None,
+        call_or_event_or_dt: ServiceCall | Event | HomeAssistant | datetime | None,
```

Comment:
This is silly, either make it an anonymous parameter: `_: Any` or call `register_webhook` with an intermediate lambda:
`async_at_started(hass, lambda hass: register_webhook(None))`

Reply:
Completely agree, but had not better alternative, now I do